.github/workflows/ci.yml

```diff
@@ -172,7 +172,9 @@ jobs:
           path: ./.coverage
     env:
       DATABRICKS_CONN_TOKEN: ${{ secrets.DATABRICKS_CONN_TOKEN }}
-      DATABRICKS_CONN_HOST: https://dbc-9c390870-65ef.cloud.databricks.com/
+      DATABRICKS_CONN_HOST: https://adb-2703548196728655.15.azuredatabricks.net/
```

Comment:
Should the host also be stored in the GitHub environment variable so that future host changes do not require code modifications?

Reply:
+1